packages/components/src/theme/navbar.scss

```diff
@@ -401,18 +401,11 @@
     border-style: none;
   }
 
-  // this nonsense is necessary because the dropdown for the find-menu looks different in
-  // firefox than in other browsers Issue 43715
-  @-moz-document url-prefix() {
-    .navbar__search-input { height: 35px; }
-  }
-
   .navbar__input-group {
       width: 320px;
   }
   .navbar__find-and-search-button {
       border-color: $white;
-      padding-bottom: 7px; // also part of styling change necessary for Issue 43715
```

Comment:
not sure which change in this file is related or that we care, but I pulled this down locally and the dropdown search selector looks good in Chrome and FF but not in Safari. In Safari, the height is slightly off for me and the magnifying glass icon isn't positioned to the left.